frontend/javascripts/test/e2e-setup.ts

```diff
@@ -78,6 +86,12 @@ global.fetch = function fetchWrapper(url, options) {
   return promise;
 };
 
+export async function scanDatasetsFromDisk() {
+  await fetch(`http://localhost:9000/data/triggers/checkInboxBlocking?token=${currToken}`, {
```

Comment:
Could this use the `triggerDatasetCheck` function from `admin_rest_api`? I’d like to avoid having another client knowing about the http api, so that we don’t have to be extra careful when changing it.